python3/src/stlib3/physics/mixedmaterial/rigidification.py

```diff
@@ -118,9 +118,12 @@ def mfilter(si, ai, pts):
                                           position=[list(allPositions[i]) for i in selectedIndices])
         rigidifiedParticules.addObject("RigidMapping", name="mapping", globalToLocalCoords=True, rigidIndexPerPoint=indicesMap)
 
-        sourceObject.removeObject(sourceObject.solver)
-        sourceObject.removeObject(sourceObject.integration)
-        sourceObject.removeObject(sourceObject.correction)
+        if "solver" in sourceObject.objects:
+            sourceObject.removeObject(sourceObject.solver)
+        if "integration" in sourceObject.objects:
+            sourceObject.removeObject(sourceObject.integration)
+        if "correction" in sourceObject:
```

Comment:
@damienmarchal we miss a `.objects` here

Reply:
OMG